pytest_splunk_addon/standard_lib/data_models/Network_Traffic.json

```diff
@@ -83,13 +83,13 @@
         },
         {
           "name": "dest_translated_ip",
-          "type": "conditional",
+          "type": "optional",
           "condition": "(protocol=IP OR protocol=ICMP) AND src_translated_ip=*",
           "comment": "The NATed IPv4 or IPv6 address to which a packet has been sent."
         },
         {
           "name": "dest_translated_port",
-          "type": "conditional",
+          "type": "optional",
```

Comment:
I see it was conditional, and the condition is actually good. So please keep as it was.

Reply:
This is marked as optional in sheet https://docs.google.com/spreadsheets/d/1ZFDC0Efn-bHvcU1Qy78s95GCfWyxt6IUhTv94j3yagk/edit?gid=1499472344#gid=1499472344 hence the change.

Reverted it to conditional

Reply:
Yes, I corrected it in the sheet. Thanks.